pyproject.toml

```diff
@@ -1,5 +1,4 @@
 [tool.black]
-skip-string-normalization = true
 exclude = '/(tests/hooks-abort-render/hooks|docs\/HelloCookieCutter1)/'
 line-length = 88
```

Comment:
Let's maybe remove the `line-length` and `target-version` parameters and just use Black's defaults

Reply:
`target-version` I think we can remove.
About `line-length` we cannot remove it because `flake8` use. Please read - https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8

Reply:
We should re-configure flake to support black, as per the documentation you sent:
```
max-line-length = 88
extend-ignore = E203
```